api/hypershift/v1beta1/openstack.go

```diff
@@ -136,6 +140,17 @@ type OpenStackPlatformSpec struct {
 	// +kubebuilder:validation:XValidation:rule="isIP(self)",message="floatingIP must be a valid IPv4 or IPv6 address"
 	// +optional
 	IngressFloatingIP string `json:"ingressFloatingIP,omitempty"`
+
+	// ImageRetentionPolicy defines the policy for handling resources associated with the image
```

Comment:
nit: start fields doc with lowercase i.e imageRetentionPolicy, so gdocs match the serialized version of the field.